ext/gd/tests/imagebmp_nullbyte_injection.phpt

```diff
@@ -0,0 +1,17 @@
+--TEST--
+Testing null byte injection in imagebmp
+--SKIPIF--
+<?php
+if(!extension_loaded('gd')) die('skip gd extension not available');
+if (!gd_info()['BMP Support']) die('skip BMP support not available');
```

Comment:
Can we safely access this index without `isset`?

Reply:
Yes. ext/gd always adds this array element.